ember-cli-build.js

```diff
@@ -65,7 +74,17 @@ module.exports = function() {
       'htmlbars-syntax':       htmlbarsPackage('htmlbars-syntax'),
       'simple-html-tokenizer': htmlbarsPackage('simple-html-tokenizer'),
       'htmlbars-test-helpers': htmlbarsPackage('htmlbars-test-helpers', { singleFile: true }),
-      'htmlbars-util':         htmlbarsPackage('htmlbars-util')
+      'htmlbars-util':         htmlbarsPackage('htmlbars-util'),
+      'glimmer-compiler':      glimmerPackage('glimmer-compiler'),
+      'glimmer-object':        glimmerPackage('glimmer-object'),
+      'glimmer-reference':     glimmerPackage('glimmer-reference'),
+      'glimmer-runtime':       glimmerPackage('glimmer-runtime'),
+      'glimmer-syntax':        glimmerPackage('glimmer-syntax'),
+      'glimmer-test-helpers':  glimmerPackage('glimmer-test-helpers'),
+      'glimmer-util':          glimmerPackage('glimmer-util'),
+      'glimmer-wire-format':   glimmerPackage('glimmer-wire-format'),
+      'glimmer':               glimmerPackage('glimmer'),
+      'simple-html-tokenizer': glimmerPackage('simple-html-tokenizer')
```

Comment:
> This is duplicated. Do we need two different versions of SHT? If not, then removing the duplicate should be fine, if we do need different ones then we need to figure out how to namespace them to prevent clobbering.

this is a @chancancode question :P